### ISSUE TITLE
vverbose don't show author in output

### ISSUE THREAD

**Issue by mr-tz** [mandiant/capa#993]
analogous to #973, remove author as not really helpful when inspecting rules

**Comment by pradkrish** [comment]
I am happy to work on this. is this still open?

**Comment by williballenthin** [comment]
that would be great! please make an attempt an open a PR for review and merge

**Comment by pradkrish** [comment]
Thanks, consider me a beginner. The usage is not really clear to me. I installed the developer version in my venv and I am able to import capa inside my venv. Can you give me an example code with an input elf file that I can use for this PR? Thanks.

**Comment by mr-tz** [comment]
you should be able to test with any PE or ELF file by running `capa -vv <sample>`

**Comment by pradkrish** [comment]
When I tried to run `capa -vv /usr/bin/pwd`

I got the following error
```
loading :   2%|█▏                                             | 16/665 [00:00<00:00, 2455.50 rules/s]
ERROR:capa:invalid rule: /home/pradeep/repos/capa/capa/../rules/communication/socket/tcp/send/obtain-transmitpackets-callback-function-via-wsaioctl.yml: invalid rule: unexpected statement: number/x64
```
I found  similar error when I tried it  on some elf files in `capa/tests/data`

**Comment by mr-tz** [comment]
:( please see related issue: https://github.com/mandiant/capa-rules/issues/551
to quickly fix you can checkout the respective rules tag

**Comment by pradkrish** [comment]
Thanks, it works now. To fix the issue here, is it about removing line `202` from `capa/render/vverbose.py`?

**Comment by mr-tz** [comment]
yeah, kinda, you can take https://github.com/mandiant/capa/pull/973/files as a good example

**Comment by pradkrish** [comment]
I have a question about the PR process. After committing the changes locally, when I run `git push origin <branch-name>`, it triggers linting and then tests which ends like this

```
Results (454.66s):
     479 passed
     129 xfailed
       3 skipped
Tests succeed!! 🎉
Stashed changes 'pre-push-1652443209' restored
SUCCEEDED 🎉🎉

```
The tests clearly pass but I don't see any updates on the forked repo.

**Comment by mr-tz** [comment]
Do you get any output on the command line?
This page has further details on creating a PR https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request and I am happy to help here as well.

**Comment by mr-tz** [comment]
@pradkrish can you submit your PR or share your changes? Would love to get this in soon.

**Comment by pradkrish** [comment]
@mr-tz That is the problem. After the push operation, I do not see it when I checked on my forked repo for the pushed branch. This is despite passing tests. Anyways, the only change I have done is to remove the line `capa/render/vverbose.py:202`. Sorry that I couldn't complete the PR. I have done this several hundred times and I don't know what I am missing here. :thinking:

**Comment by mr-tz** [comment]
Can you double-check to commit and/or push with the `--no-verify` option?
